### PR TITLE
fix: fix minor style bugs using context menu

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

SSIA

### CODE REVIEW

**Review by piggggggggy — APPROVED** [review]
LGTM!!!

File: src/services/administration/iam/user/modules/user-management-modal/UserCreateModal.vue [hunk 1]
```diff
@@ -86,7 +86,9 @@
                 <p-field-group :label="$t('IDENTITY.USER.FORM.ASSIGN_DOMAIN_ROLE')"
                                class="input-form"
                 >
-                    <p-select-dropdown v-model="formState.domainRole"
+                    <!-- CAUTION: Do not remove key binding at select dropdown. This is for initiating scroll parent to refresh fixed menu style. -->
```
Comment (yuda110) [inline src/services/administration/iam/user/modules/user-management-modal/UserCreateModal.vue:89]: question: Does this apply to other pages, too?

**Review by yuda110 — APPROVED** [review]
LGTM! I left a question.

File: src/services/administration/iam/user/modules/user-management-modal/UserCreateModal.vue (hunk 1 above)
Comment (WANZARGEN) [reply src/services/administration/iam/user/modules/user-management-modal/UserCreateModal.vue:89]: No, this page has tab in modal. That is why it is needed.

**Pull request merged**